.circleci/config.yml

```
@@ -42,6 +42,36 @@ commands:
                 command: carthage bootstrap --platform all --cache-builds --configuration Debug --use-xcframeworks   
 
 jobs:
+  detect-breaking-changes:
+    macos:
+      xcode: 13.3.0
+    steps:
+      - checkout
+      - install-mapbox-token
+      - run:
+          name: "Diagnose breaking changes"
+          command: |
+            swift package diagnose-api-breaking-changes --products MapboxDirections --baseline-dir swift-package-baseline --breakage-allowlist-path swift-package-baseline/breakage-allowlist-path.txt $(cat swift-package-baseline/baseline.txt) > diagnose-output.txt || TRUE
```

I would recommend keeping tool output in the CI logs:
```suggestion
            swift package diagnose-api-breaking-changes --products MapboxDirections --baseline-dir swift-package-baseline --breakage-allowlist-path swift-package-baseline/breakage-allowlist-path.txt $(cat swift-package-baseline/baseline.txt) | tee diagnose-output.txt || TRUE
```